src/machine.h

```diff
@@ -743,6 +743,11 @@ class machine final {
         return m_c;
     }
 
+    /// \brief Returns copy of runtime config.
```

Comment:
Not really a copy. :)

Reply:
> Code never lies, comments someties do. 
:)

Fixed, it was due to a copy paste of the above comment.